src/main/java/com/cloudant/http/interceptors/CookieInterceptor.java

```diff
@@ -161,17 +161,21 @@ private String getCookie(URL url) {
     }
 
     private boolean sessionHasStarted(InputStream responseStream) {
-        //check the response body
-        Gson gson = new Gson();
-        JsonObject jsonResponse = gson.fromJson(new InputStreamReader(responseStream), JsonObject
-                .class);
-
-        // only check for ok:true, https://issues.apache.org/jira/browse/COUCHDB-1356
-        // means we cannot check that the name returned is the one we sent.
-        return jsonResponse != null
-                && jsonResponse.has("ok")
-                && jsonResponse.get("ok").isJsonPrimitive()
-                && jsonResponse.getAsJsonPrimitive("ok").isBoolean()
-                && jsonResponse.getAsJsonPrimitive("ok").getAsBoolean();
+        try {
+            //check the response body
+            Gson gson = new Gson();
+            JsonObject jsonResponse = gson.fromJson(new InputStreamReader(responseStream,
+                    "UTF-8"), JsonObject.class);
+
+            // only check for ok:true, https://issues.apache.org/jira/browse/COUCHDB-1356
+            // means we cannot check that the name returned is the one we sent.
+            return jsonResponse != null
+                    && jsonResponse.has("ok")
+                    && jsonResponse.get("ok").isJsonPrimitive()
+                    && jsonResponse.getAsJsonPrimitive("ok").isBoolean()
+                    && jsonResponse.getAsJsonPrimitive("ok").getAsBoolean();
+        } catch (UnsupportedEncodingException e) {
+            throw new RuntimeException(e);
```

Comment:
you should re-use your comment from elsewhere `//UTF-8 should be supported on all JVMs`